Doc/library/asyncore.rst

```diff
@@ -28,6 +28,8 @@
 This module provides the basic infrastructure for writing asynchronous  socket
 service clients and servers.
 
+.. include:: ../includes/wasm-notavail.rst
```

Comment:
Likewise, this module is scheduled for removal in 3.12

Reply:
I think the notes added are useful for 3.11, or at least harmless!